crates/test-files/fixtures/printing/defs.fe

```diff
@@ -1,3 +1,11 @@
+use foo::bar as baz
```

Comment:
the double lines are a bit ugly here, but Im inclined to keep things simple and leave it alone